src/main/java/org/jabref/gui/preferences/PreferencesSearchHandler.java

```diff
@@ -89,7 +89,7 @@ private void clearSearch() {
         return filteredPreferenceTabs;
     }
 
-    private void scanInputControls(Parent parent, ArrayListMultimap<PreferencesTab, Labeled> prefsTabLabelMap, PreferencesTab preferencesTab) {
+    private static void scanInputControls(Parent parent, ArrayListMultimap<PreferencesTab, Labeled> prefsTabLabelMap, PreferencesTab preferencesTab) {
```

Comment:
Please give ths method a more meaningful name. You are scanning for label controls.

Reply:
Done!